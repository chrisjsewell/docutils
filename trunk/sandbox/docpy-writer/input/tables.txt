Testing tables
--------------

:Module: pydoctest
:Summary: test table conversion
:Author: engelbert gruber <grubert@users.sourceforge.net>
:Section Author: GruBert <grubert@users.sourceforge.net>

--------------

* Spanning cells are not supported.
* Two to five columns only AFAIK.

=====  =====  ======
  A      B    A or B
=====  =====  ======
False  False  False
True   False  True
False  True   True
True   True   True
=====  =====  ======

=== === === ===
 A   B   C   D 
=== === === ===
 1   2   3   4
 1   2   3   4
=== === === ===

=== === === === ===
 A   B   C   D   E
=== === === === ===
 1   2   3   4   5
 1   2   3   4   5
=== === === === ===
