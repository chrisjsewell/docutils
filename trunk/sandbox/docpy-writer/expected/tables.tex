\ifx\locallinewidth\undefined\newlength{\locallinewidth}\fi
\setlength{\locallinewidth}{\linewidth}


\section{Testing tables\label{-testing-tables}}
\section{\module{pydoctest} --- test table conversion}
\moduleauthor{engelbert gruber}{grubert@users.sourceforge.net}
\sectionauthor{GruBert}{grubert@users.sourceforge.net}


%___________________________________________________________________________
\hspace*{\fill}\hrulefill\hspace*{\fill}

\begin{itemize}
\item {} 
Spanning cells are not supported.

\item {} 
Two to five columns only AFAIK.

\end{itemize}
\begin{tableiii}{l|l|l}{textrm}
{
A
}
{
B
}
{
A or B
}
\lineiii{
False
}
{
False
}
{
False
}
\lineiii{
True
}
{
False
}
{
True
}
\lineiii{
False
}
{
True
}
{
True
}
\lineiii{
True
}
{
True
}
{
True
}
\end{tableiii}
\begin{tableiv}{l|l|l|l}{textrm}
{
A
}
{
B
}
{
C
}
{
D
}
\lineiv{
1
}
{
2
}
{
3
}
{
4
}
\lineiv{
1
}
{
2
}
{
3
}
{
4
}
\end{tableiv}
\begin{tablev}{l|l|l|l|l}{textrm}
{
A
}
{
B
}
{
C
}
{
D
}
{
E
}
\linev{
1
}
{
2
}
{
3
}
{
4
}
{
5
}
\linev{
1
}
{
2
}
{
3
}
{
4
}
{
5
}
\end{tablev}

